src/relay/transforms/fold_constant.cc

```diff
@@ -230,7 +216,17 @@ class ConstantFolder : public ExprMutator {
     auto entry_func = Downcast<Function>(mod->Lookup("main"));
     expr = expr.as<FunctionNode>() == nullptr ? entry_func->body : entry_func;
 
-    FInterpreter executor = GetInterpreter(mod);
+    using tvm::transform::PassContext;
```

Comment:
Why inline this back in?

Reply:
We will need the make sure the context is set during the `executor(input)` call besides the creation.